src/lib/hooks/useCurrencyLogoURIs.ts

```diff
@@ -54,15 +53,27 @@ export function getTokenLogoURI(address: string, chainId: SupportedChainId = Sup
   }
 }
 
-export default function useCurrencyLogoURIs(currency?: Currency | null): string[] {
-  const locations = useHttpLocations(currency instanceof WrappedTokenInfo ? currency.logoURI : undefined)
+export default function useCurrencyLogoURIs(
+  currency:
+    | {
+        isNative?: boolean
+        isToken?: boolean
+        address?: string
+        chainId: number
+        logoURI?: string
+      }
+    | null
+    | undefined
+): string[] {
+  const locations = useHttpLocations(currency?.logoURI)
   return useMemo(() => {
     const logoURIs = [...locations]
     if (currency) {
-      if (currency.isNative) {
+      if (currency.isNative || currency.address === 'NATIVE') {
```

Comment:
think there is a const in analytics/constants - we should ideally use it everywhere to prevent misspellings